sklearn/gaussian_process/_gpc.py

```diff
@@ -606,6 +606,10 @@ def optimizer(obj_func, initial_theta, bounds):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    GaussianProcessRegressor : Regressor module for Gaussian Proccess.
```

Comment:
```suggestion
    GaussianProcessRegressor : Gaussian process regression (GPR).
```

The easiest is just to check the first docstring line in the estimator, here https://scikit-learn.org/stable/modules/generated/sklearn.gaussian_process.GaussianProcessRegressor.html